lib/embed.dart

```diff
@@ -893,6 +914,24 @@ class Embed {
     dialog.showOk('Keyboard shortcuts', keyMapToHtml(keys.inverseBindings));
   }
 
+  WindowBase get _hostWindow {
+    if (window.parent != null) {
+      return window.parent;
+    }
+
+    return window;
+  }
+
+  void _showInstallPage() {
+    ga?.sendEvent('main', 'install');
```

Comment:
Any value to including in this call the destination page (using two actions: `install-dart` and `install-flutter`, for example)?